.github/workflows/validate-bicep.yaml

```diff
@@ -44,11 +52,23 @@ jobs:
           bicep --version
       - name: Check out repo
         uses: actions/checkout@v4
+      - name: Create a secure local registry
+        id: create-local-registry
+        uses: ./.github/actions/create-local-registry
+        with:
+          secure: "true"
+          registry-name: ${{ env.LOCAL_REGISTRY_NAME }}
+          registry-server: ${{ env.LOCAL_REGISTRY_SERVER }}
+          registry-port: ${{ env.LOCAL_REGISTRY_PORT }}
+      - name: Publish bicep types 
+        run: | 
+          bicep publish-extension ./hack/bicep-types-radius/generated/index.json --target br:${{ env.LOCAL_REGISTRY_SERVER }}:${{ env.LOCAL_REGISTRY_PORT }}/radius:latest --force  
```

Comment:
Do we need the registry? It is possible to place the extension on disk?

Reply:
We need *a* registry but it doesn't have to be a locally hosted one. The action to create one already exists in the repo (and this is how we do it in other workflows) so it made sense to use that

Reply:
Ok cool 👍  

For some reason I thought we had the ability to reference an extension as a file from `bicepconfig.json`. If that were possible, it would be fewer moving pieces.